docs_rst/installation.rst

```diff
@@ -11,11 +11,11 @@ FireWorks requires a single MongoDB instance to store and access your workflows.
 
 To install MongoDB *locally*, follow the instructions at `MongoDB <http://www.mongodb.org>`_.
 
-To access via a *cloud provider*, you might try `Mongolab <http://www.mongolab.com>`_ or search for a different one. If you are using Mongolab, here are a few notes:
+To access via a *cloud provider*, you might try `mLab <http://mlab.com/>`_ or search for a different one. If you are using mLab, here are a few notes:
 
-    * Set up an account via the Mongolab web site instructions. When asked to pick a server type (e.g. Amazon, Google, etc) you can just choose free option of 500MB. This is more than enough to get started.
-    * Mongolab will ask you to create a database; any name is fine, but make sure you write down what it is.
-    * After creating a database, note that you'll need to create at least one admin user in order to access the database.
+    * Set up an account via the mLab web site instructions. When asked to pick a server type (e.g. Amazon, Google, etc) you can just choose free option of 500MB. This is more than enough to get started.
+    * mLab will ask you to create a database; any name is fine, but make sure you write down what it is.
+    * After creating a database, note that you'll need to create at least one database user in order to access the database.
```

Comment:
Unless I missed something, the mLab interface does not mention explicitly the possibility to create an **admin** user. In order to avoid confusion readers, I thought it would be better to use the same terminology as mLab (i.e. *database user*)